60.md

```diff
@@ -0,0 +1,188 @@
+3. A user has `kind:7376` events that represent the spending history of the wallet -- This history is for informational purposes only and is completely optional.
+
+## Wallet Event
+```jsonc
+{
+    "kind": 37375,
+    "content": nip44_encrypt([
+        [ "balance", "100", "sat" ],
+        [ "privkey", "hexkey" ] // explained in NIP-61
+    ]),
+    "tags": [
+        [ "d", "my-wallet" ],
+        [ "mint", "https://mint1" ],
+        [ "mint", "https://mint2" ],
+        [ "mint", "https://mint3" ],
+        [ "name", "my shitposting wallet" ],
+        [ "unit", "sat" ],
+        [ "description", "a wallet for my day-to-day shitposting" ],
+        [ "relay", "wss://relay1" ],
+        [ "relay", "wss://relay2" ],
+    ]
+}
+```
+
+The wallet event is a parameterized replaceable event `kind:37375`.
+
+Tags:
+* `d` - wallet ID.
+* `mint` - Mint(s) this wallet uses -- there MUST be one or more mint tags.
+* `relay` - Relays where the wallet and related events can be found. -- one ore more relays SHOULD be specified. If missing, clients should follow [[NIP-65]].
+* `unit` - Base unit of the wallet (e.g. "sats", "usd", etc).
+* `name` - Optional human-readable name for the wallet.
+* `description` - Optional human-readable description of the wallet.
+* `balance` - Optional best-effort balance of the wallet that can serve as a placeholder while an accurate balance is computed from fetching all unspent proofs.
+* `privkey` - Private key used to unlock P2PK ecash. MUST be stored encrypted in the `.content` field. **This is a different private key exclusively used for the wallet, not associated in any way to the user's nostr private key** -- This is only used when receiving funds from others, described in NIP-61.
+
+Any tag, other than the `d` tag, can be [[NIP-44]] encrypted into the `.content` field.
+
+### Deleting a wallet event
+Due to PRE being hard to delete, if a user wants to delete a wallet, they should empty the event and keep just the `d` identifier and add a `deleted` tag.
+
+## Token Event
+Token events are used to record the unspent proofs that come from the mint.
+
+There can be multiple `kind:7375` events for the same mint, and multiple proofs inside each `kind:7375` event.
+
+```jsonc
+{
+    "kind": 7375,
+    "content": nip44_encrypt({
+        "mint": "https://stablenut.umint.cash",
+        "proofs": [
+            {
+                "id": "005c2502034d4f12",
+                "amount": 1,
+                "secret": "z+zyxAVLRqN9lEjxuNPSyRJzEstbl69Jc1vtimvtkPg=",
+                "C": "0241d98a8197ef238a192d47edf191a9de78b657308937b4f7dd0aa53beae72c46"
+            }
+        ]
+    }),
+    "tags": [
+        [ "a", "37375:<pubkey>:my-wallet" ]
+    ]
+}
+```
+
+`.content` is a [[NIP-44]] encrypted payload storing the mint and the unencoded proofs.
+* `a` an optional tag linking the token to a specific wallet.
```

Comment:
Why define this optional? I think this could lead to some weird issues. Imagine an user having two wallets and both picking this event. But the proofs belong to only one of the wallets.

Reply:
it doesn't matter, only one of them will be able to spend it; it's up to the client to differentiate and not list the funds as the same.

The reason it's optional is that a user might choose to completely skip the creation of a wallet or association of tokens with a specific wallet.